src/SWP/Bundle/WebRendererBundle/Theme/Configuration/TenantableConfigurationProvider.php

```diff
@@ -63,14 +63,19 @@ public function __construct(
      */
     public function getConfigurations()
     {
-        $configs = array_map(
-            [$this->loader, 'load'],
-            $this->fileLocator->locateFilesNamed($this->configurationFilename)
-        );
+        // Handle there being no theme.json file
+        try {
```

Comment:
hmm... @takeit any ideas? I don't like that try..catch and just hiding this error.